tests/kafkatest/services/streams.py

```diff
@@ -94,7 +95,7 @@ def abortThenRestart(self):
             self.logger.info("Restarting Kafka Streams on " + str(node.account))
             self.start_node(node)
 
-    def wait(self, timeout_sec=360):
+    def wait(self, timeout_sec=1440):
```

Comment:
Yeah I consciously upped this to match trunk since the new test also requires it. So this is good.